index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Home | QMart</title>
+    <link rel="stylesheet" href="https://quik-ui.netlify.app/style.css" />
+    <link rel="stylesheet" href="./qmart.css" />
+    <script
```

Comment:
I think this script tag is repeated over here 
You can remove it because you have already defined it before </body> tag

Always define it before closing </body> tag

Reply:
Done